client/src/helpers/api.js

```diff
@@ -45,4 +45,11 @@ export const loginUser = async (email, password) => {
 export const getUserProfile = () => {
   return instance.get("/user/profile");
 };
+export const getLessons = (id) => {
+  var ids = id || 4;
+  return instance.get("/api/courses/"+ids);
```

Comment:
I think you're missing a closing bracket `}` here.

Reply:
ok how to edit any missed

Reply:
how to solve this problem

Reply:
Add a `}` where it is missing.